bundles/org.openhab.binding.androidtv/src/main/java/org/openhab/binding/androidtv/internal/AndroidTVBindingConstants.java

```diff
@@ -43,15 +44,32 @@ public class AndroidTVBindingConstants {
     public static final String CHANNEL_APP = "app";
     public static final String CHANNEL_APPNAME = "appname";
     public static final String CHANNEL_APPURL = "appurl";
+    public static final String CHANNEL_APP_ICON = "appicon";
     public static final String CHANNEL_POWER = "power";
     public static final String CHANNEL_VOLUME = "volume";
     public static final String CHANNEL_MUTE = "mute";
     public static final String CHANNEL_PLAYER = "player";
+    public static final String CHANNEL_BRIGHTNESS = "brightness";
+    public static final String CHANNEL_CONTRAST = "contrast";
+    public static final String CHANNEL_SHARPNESS = "sharpness";
+    public static final String CHANNEL_TV_CHANNEL = "tvChannel";
+    public static final String CHANNEL_SEARCH_CONTENT = "searchContent";
+    public static final String CHANNEL_AMBILIGHT = "ambilight";
+    public static final String CHANNEL_AMBILIGHT_POWER = "ambilightPower";
```

Comment:
See other comment about channel naming conventions, when changed, these alsdo have to change.